crates/dojo-core/src/world/world_contract.cairo

```diff
@@ -1274,5 +1414,77 @@ pub mod world {
                 layout
             )
         }
+
+        fn write_model_ty(ref self: ContractState, model_selector: felt252, ty: @Ty) {
+            let mut serialized = ArrayTrait::new();
+            Serde::serialize(ty, ref serialized);
+
+            // TODO: clean the Vec first
```

Comment:
I hope this is something possible with the current core lib state. As `2.8` is for now in hold. Would be great if yes.

If no, an other approach is to use dojo database to store the models definitions.

Reply:
Yes, I will use our `database` interface for the moment.